data/templates/vrrp/keepalived.conf.tmpl

```diff
@@ -5,6 +5,9 @@
 global_defs {
     dynamic_interfaces
     script_user root
+{% if vrrp.global_parameters.startup_delay is vyos_defined %}
```

Comment:
Check please your PR's before adding to the stable branch
I guess it won't work as we don't have `vyos_defined` option for 1.3 at all
Thanks.

Reply:
Good catch!